Also for reference, this was the error that resulted in the script not finishing:
```
Traceback (most recent call last):
  File "./manage.py", line 88, in <module>
  File "/home/cchq/www/production/releases/2016-04-19_17.51/python_env/local/lib/python2.7/site-packages/django/core/management/__init__.py", line 385, in ex
ecute_from_command_line
  File "/home/cchq/www/production/releases/2016-04-19_17.51/python_env/local/lib/python2.7/site-packages/django/core/management/__init__.py", line 377, in ex
ecute
  File "/home/cchq/www/production/releases/2016-04-19_17.51/python_env/local/lib/python2.7/site-packages/django/core/management/base.py", line 288, in run_fr
om_argv
  File "/home/cchq/www/production/releases/2016-04-19_17.51/python_env/local/lib/python2.7/site-packages/django/core/management/base.py", line 338, in execut
e
  File "/home/cchq/www/production/releases/2016-04-19_17.51/corehq/apps/cleanup/management/commands/fix_forms_with_missing_xmlns.py", line 56, in handle
  File "/home/cchq/www/production/releases/2016-04-19_17.51/corehq/apps/cleanup/management/commands/fix_forms_with_missing_xmlns.py", line 105, in fix_xforms
  File "/home/cchq/www/production/releases/2016-04-19_17.51/corehq/apps/cleanup/management/commands/fix_forms_with_missing_xmlns.py", line 180, in <genexpr>
  File "./submodules/dimagi-utils-src/dimagi/utils/couch/database.py", line 58, in iter_docs
  File "./submodules/dimagi-utils-src/dimagi/utils/couch/bulk.py", line 92, in get_docs
  File "/home/cchq/www/production/releases/2016-04-19_17.51/python_env/local/lib/python2.7/site-packages/requests/api.py", line 109, in post
  File "/home/cchq/www/production/releases/2016-04-19_17.51/python_env/local/lib/python2.7/site-packages/requests/api.py", line 50, in request
  File "/home/cchq/www/production/releases/2016-04-19_17.51/python_env/local/lib/python2.7/site-packages/requests/sessions.py", line 468, in request
  File "/home/cchq/www/production/releases/2016-04-19_17.51/python_env/local/lib/python2.7/site-packages/requests/sessions.py", line 576, in send
  File "/home/cchq/www/production/releases/2016-04-19_17.51/python_env/local/lib/python2.7/site-packages/requests/adapters.py", line 433, in send
requests.exceptions.SSLError: [Errno 185090050] _ssl.c:340: error:0B084002:x509 certificate routines:X509_load_cert_crl_file:system lib
Unhandled exception in thread started by <bound method ConnectionReaper.__bootstrap of <ConnectionReaper(Thread-1, stopped daemon 139674032940800)>>
Traceback (most recent call last):
  File "/usr/lib/python2.7/threading.py", line 524, in __bootstrap
    self.__bootstrap_inner()
  File "/usr/lib/python2.7/threading.py", line 564, in __bootstrap_inner
    (self.name, _format_exc()))
  File "/usr/lib/python2.7/traceback.py", line 241, in format_exc
    return ''.join(format_exception(etype, value, tb, limit))
  File "/usr/lib/python2.7/traceback.py", line 141, in format_exception
    list = list + format_tb(tb, limit)
  File "/usr/lib/python2.7/traceback.py", line 76, in format_tb
    return format_list(extract_tb(tb, limit))
  File "/usr/lib/python2.7/traceback.py", line 101, in extract_tb
    line = linecache.getline(filename, lineno, f.f_globals)
  File "/home/cchq/www/production/releases/2016-04-19_17.51/python_env/lib/python2.7/linecache.py", line 14, in getline
  File "/home/cchq/www/production/releases/2016-04-19_17.51/python_env/lib/python2.7/linecache.py", line 40, in getlines
  File "/home/cchq/www/production/releases/2016-04-19_17.51/python_env/lib/python2.7/linecache.py", line 108, in updatecache
TypeError: 'NoneType' object is not callable
```

Don't know quite what caused it, but it crashed around the time a deploy was finishing, so maybe it had something to do with that)

--- a/corehq/apps/cleanup/tests/test_fix_forms_with_missing_xmlns.py
+++ b/corehq/apps/cleanup/tests/test_fix_forms_with_missing_xmlns.py
@@ -125,7 +125,7 @@ class TestFixFormsWithMissingXmlns(TestCase, TestXmlMixin):
         form, xform = self.build_normal_app()
         self._refresh_pillow()
 
-        call_command('fix_forms_with_missing_xmlns', 'log.txt')
+        call_command('fix_forms_with_missing_xmlns', '/dev/null', 'log.txt')
 
         with open("log.txt") as log_file:
             log = log_file.read()
@@ -136,7 +136,7 @@ class TestFixFormsWithMissingXmlns(TestCase, TestXmlMixin):
         good_form, bad_form, good_xform, bad_xforms = self.build_app_with_bad_form()
         self._refresh_pillow()
 
-        call_command('fix_forms_with_missing_xmlns', 'log.txt')
+        call_command('fix_forms_with_missing_xmlns', '/dev/null', 'log.txt')
 
         with open("log.txt") as log_file:
             log = log_file.read()
@@ -150,7 +150,7 @@ class TestFixFormsWithMissingXmlns(TestCase, TestXmlMixin):
         form, good_build, bad_build, good_xform, bad_xform = self.build_app_with_recently_fixed_form()
         self._refresh_pillow()
 
-        call_command('fix_forms_with_missing_xmlns', 'log.txt')
+        call_command('fix_forms_with_missing_xmlns', '/dev/null', 'log.txt')
 
         with open("log.txt") as log_file:
             log = log_file.read()

